src/openfermion/chem/molecular_data.py

```diff
@@ -173,6 +173,45 @@ def geometry_from_file(file_name):
                 geometry += [(atom, coordinates)]
     return geometry
 
+def antisymint(two_body_integrals):
+    """
+    Args:
+    two_body_integrals : Numpy array of two-electron integrals with OpenFermion
```

Comment:
elaborate that this should be spin-orbital integrals.  not spatial.

Reply:
They are now in spatial orbital basis, but I can put them in spin-orbital basis if that is better. I suggest then a separate function of the molecular_data class dedicated to going from spatial to spin-orbital basis (that we can also use in the get_molecular_hamiltonian function).